jvm-runtime/ftl-runtime/common/runtime/pom.xml

```diff
@@ -28,6 +28,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-context-propagation</artifactId>
```

Comment:
Why is this included? This is for propagating context between multiple threads in a single request, which I don't know if we are really concerned with.